contrib/fsst-cmake/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+option(ENABLE_FSST "Enable FSST (Fast Static Symbol Table)" ${ENABLE_LIBRARIES})
+
+if (NOT ENABLE_FSST)
```

Comment:
Note to myself: the AVX512 detection at the beginning of fsst_avx512.cpp looks x86-specific. Test what happens on ARM, possibly exclude fsst_avx512.cpp from compilation on non-x86 platforms.